cypress/integration/basic-actions.spec.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+/* tslint:disable */
+
+describe("When user clicks “I can’t do that thing today.”", () => {
+    it("Opens an confirm", () => {
+        cy.visit("localhost:3000");
+        const stub = cy.stub();
+        cy.on("window:confirm", stub);
+        cy.get('button.skipButton').click().then(() => {
+          expect(stub.getCall(0)).to.be.calledWith("Are you sure you want a new thing?");
+        });
```

Comment:
You might be able to do something like this which would let use remove the class name reference: 

```suggestion
        cy.contains('I can't do this today.').click().then(() => {
          expect(stub.getCall(0)).to.be.calledWith("Are you sure you want a new thing?");
        });
```

Reference: https://docs.cypress.io/api/commands/click.html#Syntax

Reply:
Cool. Yeah, I like that as a solution for overcoming utility-based CSS. My brain was treating this selection function as restricted to CSS, which of course wouldn’t have a “contains” option. Nice that Cypress has that—at least until we decide to change the wording.